core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -1055,15 +1067,15 @@ object QualificationAppInfo extends Logging {
       pluginTypeChecker: PluginTypeChecker,
       reportSqlLevel: Boolean,
       mlOpsEnabled: Boolean,
-      penalizeTransitions: Boolean): Either[String, QualificationAppInfo] = {
+      penalizeTransitions: Boolean): Either[FailureApp, QualificationAppInfo] = {
     try {
-        val app = new QualificationAppInfo(Some(path), Some(hadoopConf), pluginTypeChecker,
-          reportSqlLevel, false, mlOpsEnabled, penalizeTransitions)
-        logInfo(s"${path.eventLog.toString} has App: ${app.appId}")
-        Right(app)
-      } catch {
-        case e: Exception =>
-          Left(handleException(e, path))
-      }
+      val app = new QualificationAppInfo(Some(path), Some(hadoopConf), pluginTypeChecker,
+        reportSqlLevel, false, mlOpsEnabled, penalizeTransitions)
+      logInfo(s"${path.eventLog.toString} has App: ${app.appId}")
+      Right(app)
+    } catch {
+      case e: Exception =>
```

Comment:
nit: Unintentional spaces?

Reply:
It looked like the indentation was off by 2, so I moved the code block. Correct me if I was wrong.